hack/jenkins/e2e.sh

```diff
@@ -639,8 +641,11 @@ case ${JOB_NAME} in
     : ${E2E_TEST:="false"}
     : ${USE_KUBEMARK:="true"}
     : ${NUM_MINIONS:="10"}
+    KUBEMARK_MASTER_SIZE="n1-standard-4"
```

Comment:
I think that we should explicitly set MASTER_SIZE and MINION_SIZE here too (as in the other suite) to be independent from default configuration...